### PR TITLE
Initial release of rules_gazebo

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (rules_gz) have been updated in this PR. Please review the changes.

**Comment by Wyverald** [comment]
this is potentially a long shot, but is there any chance you could rename this project to `rules_gazebo`? `gz` is the default suffix for gzipped data, so `rules_gz` could be very misleading.

**Comment by mjcarroll** [comment]
Didn't think about the potential conflict with `gz`.  I think renaming is the right move here.  Will do that now.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (rules_gazebo) have been updated in this PR. Please review the changes.

**Comment by mjcarroll** [comment]
The package is now named `rules_gazebo`. Thank you for the feedback.

**Review by fmeum — DISMISSED** [review]

File: modules/rules_gazebo/0.0.1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"rules_gazebo"
+
+module(
+    name = "rules_gazebo",
+    version = "0.0.1",
+    compatibility_level = 1,
+)
+
```
Comment (fmeum) [inline modules/rules_gazebo/0.0.1/MODULE.bazel:8]: Needs a `bazel_dep` for `rules_license`.

File: modules/rules_gazebo/0.0.1/MODULE.bazel [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "6.1.2")
+bazel_dep(name = "rules_cc", version = "0.0.10")
```
Comment (fmeum) [inline modules/rules_gazebo/0.0.1/MODULE.bazel:11]: rules_cc isn't compatible with Bazel 6 in its latest version (I sent https://github.com/bazelbuild/rules_cc/pull/248 to change this). Could you try to downgrade the version? It's generally better to depend on lower versions if possible (for a library/ruleset) to allow consumers more flexibility.

File: modules/rules_gazebo/0.0.1/MODULE.bazel (hunk 1 above)
Comment (mjcarroll) [reply modules/rules_gazebo/0.0.1/MODULE.bazel]: Done in version 0.0.2

File: modules/rules_gazebo/0.0.1/MODULE.bazel (hunk 2 above)
Comment (mjcarroll) [reply modules/rules_gazebo/0.0.1/MODULE.bazel]: Downgraded to 0.0.9 in version 0.0.2

Also added non-module workflow test in upstream.

**Comment by Wyverald** [comment]
please fix the check failures.

**Review by Wyverald — APPROVED** [review]

**Pull request merged**